java/code/src/com/redhat/rhn/domain/org/usergroup/UserGroupFactory.java

```diff
@@ -158,10 +148,9 @@ public static void delete(ExtGroup extGroup) {
      * @return external group object
      */
     public static UserExtGroup lookupExtGroupByLabel(String labelIn) {
-        Map<String, String> params = new HashMap<>();
+        Map<String, Object> params = new HashMap<>();
```

Comment:
can this also be changed to use map.of?

Reply:
Sure... I probably overlooked some, and when there were more than 2 keys I didn't touch them: not sure which is more readable.